src/main/java/com/noroff/mefit/data/model/Program.java

```diff
@@ -0,0 +1,27 @@
+package com.noroff.mefit.data.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "program", schema = "public")
+public class Program {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column
+    private String name;
+
+    @Column
+    private String category;
```

Comment:
According to Schema.sql:
```suggestion
    @NotNull
    @Size(max = 50)
    @Column(nullable = false)
    private String name;

    @Column
    @Size(max = 100)
    private String category;
```